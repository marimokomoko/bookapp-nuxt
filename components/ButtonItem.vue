<template>
  <v-row class="button">
    <v-col>
      <v-btn min-width="110" :to="url" :color="color">
        {{ buttonText }}
      </v-btn>
    </v-col>
  </v-row>
</template>
<script>
export default {
  props: {
    buttonText: {
      type: String,
      required: true,
    },
    url: {
      type: String,
      required: true,
    },
    color: {
      type: String,
    },
  },
};
</script>
