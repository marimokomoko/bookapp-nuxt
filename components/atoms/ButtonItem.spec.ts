import { mount } from '@vue/test-utils'
import ButtonItem from '@/components/atoms/ButtonItem.vue'
describe('画面レイアウト', ()=>{
    it('スナップショットテスト', ()=>{
        const wrapper = mount(ButtonItem)
        expect(wrapper.html()).toMatchSnapshot()
    })
})