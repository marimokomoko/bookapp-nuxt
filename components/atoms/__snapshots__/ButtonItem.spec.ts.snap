// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`画面レイアウト スナップショットテスト 1`] = `
"<v-row class=\\"button\\">
  <v-col>
    <v-btn min-width=\\"110\\">

    </v-btn>
  </v-col>
</v-row>"
`;
